packages/vue-labs/src/components/FTable/ITableText.vue

```diff
@@ -12,12 +13,21 @@ const { row, column } = defineProps<{
 }>();
 
 const model = ref("");
+const inEdit = ref(false);
 const validity = ref<Pick<ValidityEvent, "isValid" | "validationMessage" | "validityMode">>({
     isValid: true,
     validationMessage: "",
     validityMode: "INITIAL",
 });
 const hasError = computed(() => validity.value.validityMode === "ERROR");
+const divClasses = computed(() => {
+    return {
+        "table-ng__editable": true,
+        "table-ng__editable__numeric": column.tnum,
+        "table-ng__editable__align-right": column.align === "right",
```

Comment:
```suggestion
        "table-ng__align-right": column.align === "right",
```

?

Tänker att detta ska väl slå även på icke-editerbara / visningsläget?